udata_gouvfr/templates/openfield16.html

```diff
@@ -122,6 +122,23 @@
                     <dt>19h</dt><dd>Cocktail de clôture</dd>
                 </dl>
 
+
+                <h2>Les données mises en open data dans le cadre d’#openfield16</h2>
+
+                <p>Dans le cadre du hackathon #openfield16, le ministère de la ville, de la jeunesse et des sports met à disposition en open data les jeux de données suivants :</p>
+
+                <ul>
+                    <li><a href="https://www.data.gouv.fr/fr/datasets/referentiel-dindicateurs-pour-auto-evaluer-leco-responsabilite-devenements-sportifs/">Référentiel d'indicateurs pour autoévaluer l'éco-responsabilité d'événements sportifs</a></li>
```

Comment:
Oh, I misunderstood your point: yes the dynamic list is still here (there's a specific badge) but this paragraph is just a notice about the new dataset published for openfield16.
I think we can improve this list with the embeds, but it may be weird to have 2 different kind of card on the same page.